psutil/_psutil_common.c

```diff
@@ -170,6 +170,15 @@ convert_kvm_err(const char *syscall, char *errbuf) {
 }
 #endif
 
+// ====================================================================
+// --- macOS
+// ====================================================================
+
+#ifdef PSUTIL_OSX
+#include <mach/mach_time.h>
+
+struct mach_timebase_info MACH_TIMEBASE_INFO;
+#endif
```

Comment:
In order to be consistent with `PSUTIL_SYSTEM_INFO`, `PSUTIL_CRITICAL_SECTION` and `PSUTIL_DEBUG` I suggest to:

* call this `PSUTIL_MACH_TIMEBASE_INFO`
* define it in `_psutil_common.h` file

Reply:
> In order to be consistent with `PSUTIL_SYSTEM_INFO`, `PSUTIL_CRITICAL_SECTION` and `PSUTIL_DEBUG` I suggest to:
> 
>     * call this `PSUTIL_MACH_TIMEBASE_INFO`

That makes more sense indeed. Sorry.

>     * define it in `_psutil_common.h` file

I don't understand this. The header file does have the extern definition and storage can only be in the .c file. What am I missing?